semanticdb/scalac/library/src/main/scala-2.12.17/scala/meta/internal/semanticdb/scalac/SemanticdbReporter.scala

```diff
@@ -19,15 +19,20 @@ class SemanticdbReporter(underlying: Reporter)
       case 2 => underlying.error(pos, msg)
       case _ =>
     }
-
   }
 
   override def hasErrors: Boolean = underlying.hasErrors
 
   override def hasWarnings: Boolean = underlying.hasWarnings
 
-}
 
+  // overriding increment is enough so make sure that error/warning
+  // counts are the same as in underlying reporter
+   override def increment(severity: Severity): Unit = {
+    super.increment(severity)
+    underlying.increment(severity)
+  }
+}
```

Comment:
This does not appear to be sufficient. This did not fix my issues with Silencer while the full Scala 2.13 version of `SemanticdbReporter.scala` works.